Assets/ConsentManagementProvider/Plugins/iOS/Source/SwiftBridge.swift

```diff
@@ -104,6 +125,7 @@ import UIKit
                 campaigns: SPCampaigns(
                     gdpr: gdpr ? SPCampaign(targetingParams: gdprTargetingParams, groupPmId: gdprPmId) : nil,
                     ccpa: ccpa ? SPCampaign(targetingParams: ccpaTargetingParams, groupPmId: ccpaPmId) : nil,
+                    usnat: usnat ? SPCampaign(targetingParams: usnatTargetingParams, groupPmId: usnatPmId, transitionCCPAAuth: config.transitionCCPAAuth, supportLegacyUSPString: config.supportLegacyUSPString) : nil,
```

Comment:
Hey @andresilveirah ! 
Could you please judge me on this - I think I've made a mistake here (lines 126, 127, 128). 
Pay attention to me using `<campaignPmId>` as `groupPmId` (example: `groupPmId: usnatPmId`) for every single campaign. 
I have a strong feeling that's incorrect. But nobody noticed so far since we do not provide `groupPmId` feature within Unity SDK. 
Should it be: (1) `groupPmId: campaignPmId` OR (2) `groupPmId: null` OR should I add `groupPmId/childPmId` feature to Unity SDK (It has not been requested yet)?

Reply:
@Nevazhnovu none of the above :D 

`groupPmId` should be the group pm id of that legislation (which may or may not be the same as the regular pm id), ie.
```swift
SPCampaigns(gdpr: SPCampaign(groupPmId: gdprGroupPmId))
```

However, I think we can leave that as `null` since it hasn't been required.